lib/LedgerSMB/Template/XLSX.pm

```diff
@@ -65,7 +65,8 @@ my $rowcount;
 my $currcol;
 
 sub _worksheet_handler {
-    $rowcount = -1;
+    $rowcount = 0;
+    -- $rowcount;
```

Comment:
@ziuq absent a response by @ylavoie, I think we can remove these variables entirely. They're not used and setting to zero end decrementing is just too ugly for words for a variable which isn't adding value.

(I'm referring to both `$rowcount` and `$currcol`.)